README.md

```diff
@@ -15,6 +15,8 @@ Options:
 - generate changelogs for some supermarket hosted cookbooks
 - generate changelogs for all git located cookbooks
 
+This plugin works in policyfile style repositories or classical repositories with a Berksfile
```

Comment:
OPT: Maybe add specific comment that you rely on the policy file being named Policyfile.rb

Reply:
I've published a patch after this one adding support for --policy option that control the policyfile path used by knife-changelog.